source/ghost-text.css

```diff
@@ -12,3 +12,10 @@
 [data-gt-field='enabled'] {
 	box-shadow: #00acee 0 0 20px 5px inset !important;
 }
+.field {
+	display: table;
```

Comment:
This style is injected into the host, not in the options. I don’t think there's any advantage in using `display:table` here because the form won't look like a table. How is it different than without this?  Can you include a screenshot of the options?

Reply:
The advantage is to achieve this multiline proper alignment of the checkboxes

<img width="369" alt="gt3" src="https://user-images.githubusercontent.com/12956286/222367511-c601db38-a896-48e5-9c59-dd924d83596e.png">

to avoid this

<img width="369" alt="gt1" src="https://user-images.githubusercontent.com/12956286/222367528-38e90ead-395e-410f-b33d-a4a4bab19bd9.png">

and also avoid this (that's where table helps)
<img width="369" alt="gt2" src="https://user-images.githubusercontent.com/12956286/222367518-18584a04-5667-4113-bc4c-0a6113c5c43d.png">


But you're right about the injection, think deleted the import by mistake. Will anyway move it to a separate options.css maybe?

Reply:
move to options.css with an explicit import

Reply:
Why does your page look like that though? The font should be sans serif. Are you testing it in Chrome?

This is what I see on the store currently:

<img width="430" alt="Screenshot" src="https://user-images.githubusercontent.com/1402241/222381788-cf43be5e-68cf-4a74-80a3-0da96ea0cd4f.png">

Reply:
These screenshots are from opening the html directly since it was more convenient to test (e.g., you can change window width there, but they also look), but it doesn't matter, here are the extension options with `.field{display:table}` removed

<img width="373" alt="gt22" src="https://user-images.githubusercontent.com/12956286/222383497-69f354c0-2252-44ca-bcbe-7df0b2f50e47.png">